homework_2/src/serviceWorker.js

```diff
@@ -0,0 +1,141 @@
+// This optional code is used to register a service worker.
```

Comment:
Do you really need this service  worker here, or it was auto generated as well? :)